test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1

```diff
@@ -66,7 +66,9 @@ Describe "Get-ChildItem" -Tags "CI" {
             $file.Count | Should be 1
             $file.Name | Should be $item_F
         }
-        It "Should continue enumerating a directory when a contained item is deleted" {
+        # Test is pending on Unix platforms because of a behavior change in the latest .NET Core.
```

Comment:
The behavior change was by design, to better align with Full .NET. So yes, this test and the corresponding fix need to be revisited. Here is the tracking issue: #4145